articles/openshift/oshift-how-use-portworx-shared-volumes.md

```diff
@@ -0,0 +1,238 @@
+---
+title: How to use Portworx shared volumes | UKCloud Ltd
+description: Explains how to use shared volumes in a Portworx integrated OpenShift cluster
+services: openshift
+author: Ben Bacon
+reviewer:
+lastreviewed: 16/10/2019
+
+toc_rootlink: How To
+toc_sub1: Use Portworx with OpenShift
+toc_sub2:
+toc_sub3:
+toc_sub4:
+toc_title: Use Portworx shared volumes
+toc_fullpath: How To/Use Portworx with OpenShift/oshift-how-use-portworx-shared-volumes.md
+toc_mdlink: oshift-how-use-portworx-shared-volumes.md
+---
+
+# How to use Portworx shared volumes
+
+## Overview
+
+Portworx is a cloud-native storage solution that is now available as an integration with our OpenShift offering. This article describes how to create shared volumes.
+
+### Intended audience
+
+This article assumes you have access to a Portworx integrated OpenShift 3.11 cluster and that you have cluster-admin rights. It also assumes familiarity with `oc`, the OpenShift command-line client. 
```

Comment:
Should we specify 3.11 or greater? That way we future proof it and only have to validate that this hasn't changed how these features are used (which I don't think a new version of OpenShift would change).